docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -151,19 +151,9 @@ resource "docker_container" "workspace" {
 }
 ```
 
-For more targeted control, specify which attributes to ignore:
-
-```hcl
-resource "docker_container" "workspace" {
-  lifecycle {
-    ignore_changes = [name]
-  }
-
-  count = data.coder_workspace.me.start_count
-  name  = "coder-${data.coder_workspace_owner.me.name}-${lower(data.coder_workspace.me.name)}"
-  ...
-}
-```
+Try to keep `ignore_changes` scoped to just the fields which were called out in the notification. Excessive use of
+`ignore_changes` can lead to undesirable outcomes in Terraform, like innocuous changes (which wouldn't cause replacements)
+to be ignored.
```

Comment:
> scoped to just the fields which were called out in the notification

we're introducing this (L140) with `To prevent this, add a`, so our hope is that people will use this section before they encounter the notification. If it's more likely that people will only come here after an error, we should reframe this section as a way to resolve that.

```suggestion
Limit the scope of `ignore_changes` to include only the fields specified in the notification.
If you include too many fields, Terraform might ignore changes that wouldn't otherwise cause drift.
```

Reply:
Good suggestion, thanks 👍